build.gradle

```diff
@@ -71,12 +71,14 @@ ext.libraries = [
         rxJava                         : 'io.reactivex:rxjava:1.0.14',
 
         // Parts of StorIO
-        storIOCommon                   : project(':storio-common'),
-        storIOSQLite                   : project(':storio-sqlite'),
-        storIOContentResolver          : project(':storio-content-resolver'),
-        storIOSQLiteAnnotation         : project(':storio-sqlite-annotations'),
-        storIOSQLiteAnnotationProcessor: project(':storio-sqlite-annotations-processor'),
-        storIOTestCommon               : project(':storio-test-common'),
+        storIOCommon                             : project(':storio-common'),
+        storIOSQLite                             : project(':storio-sqlite'),
+        storIOContentResolver                    : project(':storio-content-resolver'),
+        storIOSQLiteAnnotation                   : project(':storio-sqlite-annotations'),
+        storIOSQLiteAnnotationProcessor          : project(':storio-sqlite-annotations-processor'),
+        storIOContentResolverAnnotation          : project(':storio-content-resolver-annotations'),
```

Comment:
Can you please rename it to `storIOContentResolverAnnotations` "s" at the end, my bad that I forgot to do it for sqlite annotations